### PR TITLE
all products page

### PR DESCRIPTION
**Pull request by i-am-a-codegirl**
Kindly review my all products page for the eCommerce website

The page has the following features

1. Nav : has brand name and links to home, all-products, login, cart and wishlist page
2. Filters to sort by price, ratings etc
3. Cards sections containing all available products
4. Footer : contains links to login, all-products, category. etc

image

image

image

### CODE REVIEW

File: all products/all-products.css [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    accent-color: var(--secondary-color);
+    font-size: 5rem;
+
+}
+
+.product-info{
+
+    display: flex;
+    flex-direction: column;
+ justify-content: center;
```
Comment (khushijohri2001) [inline all products/all-products.css]: alignment problem here

**Review by khushijohri2001 — APPROVED** [review]
The code looks clean and good to merge

File: all products/all-Products.html [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../style.css">
+    <link rel="stylesheet" href="./all-products.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <script src="https://kit.fontawesome.com/e45a62e93a.js" crossorigin="anonymous"></script>
+    <title>Document</title>
```
Comment (Ayesha-7) [inline all products/all-Products.html]: Suggestion: can add relevant title here

File: all products/all-Products.html [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+          <li>302017</li>
+          <li><a href=""><i class="fa-brands fa-instagram-square"></i></a></li>
+        </ul>
+        
+        </footer>
+
+    </div>
+
+
+
```
Comment (Ayesha-7) [inline all products/all-Products.html:322]: Suggestion: code can be indented/ formatted properly that makes it more readable. you can use some formatting extension for this.

File: all products/all-products.css [hunk 4]
```diff
@@ -0,0 +1,176 @@
+.products{
+    display: grid;
+
+}
+
+.products h2{
+    text-align: center;}
```
Comment (Ayesha-7) [inline all products/all-products.css:6]: Suggestion: code can be indented/ formatted properly that makes it more readable. you can use some formatting extension for this.

File: all products/all-Products.html [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+                     
+                       
+                      </div>
+                    </div>
+                  </div>
+        
+                  <div class="card">
+                    <div class="image-div">
+                      <div class="overlay-image">
+                        <img class="card-image" src="../images/coco oats.jfif" alt="error" />
```
Comment (Ayesha-7) [inline all products/all-Products.html:258]: Suggestion: can add relevant alternate text in alt attribute.

File: all products/all-Products.html [hunk 6]
```diff
@@ -0,0 +1,324 @@
...
+                     
+                       
+                      </div>
+                    </div>
+                  </div>
+                  
+                  <div class="card">
+                    <div class="image-div">
+                      <div class="overlay-image">
+                        <img class="card-image" src="../images/jar2.jpg" alt="error" />
```
Comment (Ayesha-7) [inline all products/all-Products.html:233]: Suggestion: can add relevant alternate text in alt attribute.

File: all products/all-Products.html [hunk 7]
```diff
@@ -0,0 +1,324 @@
...
+                     
+                       
+                      </div>
+                    </div>
+                  </div>
+        
+                  <div class="card">
+                    <div class="image-div">
+                      <div class="overlay-image">
+                        <img class="card-image" src="../images/enregy bar2.jpg" alt="error" />
```
Comment (Ayesha-7) [inline all products/all-Products.html:207]: Suggestion: can add relevant alternate text in alt attribute.

File: all products/all-Products.html [hunk 8]
```diff
@@ -0,0 +1,324 @@
...
+
+
+
+     <div class="bestsellers">
+            <div class="bestsellers-card">
+                <div class="card">
+                    <div class="image-div">
+                      <div class="overlay-image">
+                        <img class="card-image" src="../images/eat 3.jpg" alt="error" />
+                        <div class="card-badge-product-pg color-pink"><i class="fas fa-heart" class=""></i></div>
```
Comment (Ayesha-7) [inline all products/all-Products.html:78]: can remove extra `class` in `i` tag.

**Review by Ayesha-7 — COMMENTED** [review]
code is clean & I liked the simple UI.
Left some suggestions please check those.

**Comment by i-am-a-codegirl** [comment]
> code is clean & I liked the simple UI. Left some suggestions please check those.

For sure and thanks :)

**Pull request merged**